src/machine/lib_machine.rs

```diff
@@ -235,6 +235,10 @@ impl Machine {
             called: false,
         }
     }
+
+    pub fn cleanup_query_iter(&mut self) {
+        self.trust_me()
+    }
 }
```

Comment:
```suggestion
     }
 }
```

The cleanup is already performed on drop.